src/server/injected/injectedScript.ts

```diff
@@ -456,69 +523,13 @@ export class InjectedScript {
     range.selectNodeContents(element);
     const selection = element.ownerDocument.defaultView!.getSelection();
     if (!selection)
-      return 'error:notvisible';
+      return 'error:notconnected';
```

Comment:
I guess this might also happen in case of selecting text in an element with `user-select: none;`